src/hooks/useProductData.js

```diff
@@ -0,0 +1,8 @@
+import { useContext } from "react";
```

Comment:
This hook is not used anywhere in the project. You can remove it, if you are thinking to use it in the future you can place this file in .gitignore.